Gopkg.lock

```diff
@@ -7,6 +7,12 @@
   packages = ["quantile"]
   revision = "4c0e84591b9aa9e6dcfdf3e020114cd81f89d5f9"
 
+[[projects]]
+  branch = "session-handles"
+  name = "github.com/cbroglie/crypto11"
```

Comment:
We can revert back to using `github.com/thalesignite/crypto11` since they merged your PR.